src/Core/PackageKitBackend.vala

```diff
@@ -42,6 +42,36 @@ public class AppCenterCore.PackageKitBackend : Backend, Object {
 
     public bool working { get; private set; }
 
+    // The aptcc backend included in PackageKit < 1.1.10 wasn't able to support multiple packages
+    // passed to the search_names method at once. If we have a new enough version we can enable
+    // some optimisations when looking up packages
+    public static bool supports_parallel_package_queries {
+        get {
+            if (control.backend_name != "aptcc") {
```

Comment:
If `backend_name != "aptcc"` should it really return true or false here?

Reply:
It should return true because other backends support parallel queries fine. There was a bug in aptcc that has been fixed in the newer versions.